### ISSUE TITLE
Codelist (V 2.1) - CriteriaTaxonomy - EO_DATA.SHELTERED_WORKSHOP

### ISSUE THREAD

**Issue by JachymHercher** [OP-TED/ESPD-EDM#220]
In the CriteriaTaxonomy codelist, there is the following code:

```
CRITERION.OTHER.EO_DATA.SHELTERED_WORKSHOP
Name: EO is a sheltered workshop
Description: Only in case the procurement is reserved: is the economic operator a sheltered workshop, a 'social business' or will it provide for the performance of the contract in the context of sheltered employment programmes?
```
1.  This is similar to eForms BT-71 (Reserved Procurement). However, BT-71 is more detailed and also does not mention a "social business"; but includes "organisations pursuing a public service mission". Would it be possible to add values to the codelist so that these three can be referred to individually and resolve the difference comparing the "social business"? This would allow reuse in eForms. 

eForms BT-71 draft values:
- The contract is reserved to sheltered workshops and economic operators aiming at   the social and professional integration of disabled or disadvantaged persons.
- The execution of the contract is restricted to the framework of  sheltered employment programmes.
- _Participation in the procedure is reserved to organisations  pursuing a public service mission and fulfilling other relevant conditions in the legislation._ (This is based on Article 77(2) of Directive 2014/24/EU and Article 94(2) of Directive 2014/25/EU). 

2. I think it's a bit unfortunate that the name of the code is "sheltered workshop", but then the definition covers three concepts. (Splitting it in line with point 1 would solve that.)

(FYI @muricna)

**Comment by mireiaBM** [comment]
This issue will be addressed in version 3.0.0 of the ESPD data model.

**Comment by JachymHercher** [comment]
When addressing the issue, please make sure to align it with the latest version of eForms (there have been changed e.g. following https://github.com/eForms/eForms/issues/296#issuecomment-475210445).

**Comment by mireiaBM** [comment]
Yes, the idea with version 3.0.0 is that it will be aligned with eForms

**Comment by hricolor** [comment]
After present this issue in the last OUC of the 1st of October, and discuss with the stakeholders, this is the proposed solution that will be implemented:

Since this issue is related to the agreed solution for the issue #124, this criteria will be kept and shown only if the CA indicates in the CN that participation is reserved, meaning that only tenderers with specific social missions may tender.

This issue will be implemented within the next ESPD-EDM 3.0.0.

**Comment by psotofer** [comment]
As commented before, this issue was included in the last ESPD-EDM Release v3.0.0. therefore we proceed to close this issue.
In case you need further detail regarding this topic, please open a new issue referencing this one. 

The EO_DATA.SHELTERED_WORKSHOP concept remains in the criterion code list and will only be used if the CA indicates in the CN that participation is reserved.
